app/forms/admin/claims_filter_form.rb

```diff
@@ -74,6 +74,25 @@ def claims
         Claim.includes(:decisions).failed_bank_validation.awaiting_decision
       when "awaiting_provider_verification"
         Claim.by_policy(Policies::FurtherEducationPayments).awaiting_further_education_provider_verification.awaiting_decision
+      when "awaiting_claimant_data"
+        Claim
+          .by_policy(Policies::EarlyYearsPayments)
+          .where(submitted_at: nil)
+          .awaiting_decision
+      when "awaiting_retention_period_completion"
+        Claim
+          .by_policy(Policies::EarlyYearsPayments)
+          .joins(:early_years_payment_eligibility)
+          .where.not(submitted_at: nil)
+          .awaiting_decision
```

Comment:
If we don't want to add a belongs_to to a specific policy to the claim we could do something like
```suggestion
Claim
  .where(
    id: Policies::EarlyYearsPayments
      .select("claims.id")
      .joins(:claim)
      .where("start_date > ?", Policies::EarlyYearsPayments::RETENTION_PERIOD.ago)
  )
  .where.not(submitted_at: nil)
  .awaiting_decision
```

Not sure what's better 🤷

Reply:
I get you and agree you could go either way as may be we don't want to clutter with subtype implementation in `Claim` on the other hand may need to duplicate elsewhere. I'm leaning towards leaving this for now then later if it gets worse extract somehow...